### PR TITLE
Refactoring of some converters to always initialize variables

### LINKED ISSUE #432: NPE in some converters when calculating updated attributes
Caused by #427 and #409 

I'm experiencing a NPE in the method `ZigBeeConverterMeasurementPower.attributeUpdated(...)` thrown by this statement:
```
BigDecimal valueInWatt = BigDecimal.valueOf(value * multiplier / divisor);
```
The reason is that `multiplier` and `divisor` are both null. This happens because since #427 these variables are initialized in the method `initializeDevice()` which since #409 is only invoked on the very first initialization and then only `initializeConverter()` will be called on startup. Such variables should be better initialized in the method `initializeConverter()`.

Other converters which currently use to initialize variables in `initializeDevice()` will need to be changed too (didn't check them all).

### PR DESCRIPTION
**Pull request by TomTravaglino**
Resolves #432 

Some converters used to initialize variables on device initialization in
order to handle attributes updates. After the initilization has been
been split into a device and a converter initialization these variables
were no longer initialized when only the converter was initialized. This
made the initialization run into NPEs. This commit ensures that the
initialization is properly done when only the converter is initialized.

Signed-off-by: Tommaso Travaglino <tommaso.travaglino@telekom.de>

### CODE REVIEW

File: org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterColorTemperature.java [hunk 1]
```diff
@@ -62,24 +62,7 @@ public boolean initializeDevice() {
...
-
-        // Low Mired values correspond to high Kelvin values, hence the min Mired value yields the max Kelvin value
-        if (minTemperatureInMired == null) {
-            kelvinMax = DEFAULT_MAX_TEMPERATURE_IN_KELVIN;
-        } else {
-            kelvinMax = miredToKelvin(minTemperatureInMired);
-        }
-
-        kelvinRange = kelvinMax - kelvinMin;
+        determineMinMaxTemperature(serverClusterColorControl);
```
Comment (cdjackson) [inline org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterColorTemperature.java]: Why is this needed here? kelvinRange is not used in this method.

File: org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterMeasurementPower.java [hunk 2]
```diff
@@ -71,12 +71,8 @@ public boolean initializeDevice() {
             return false;
         }
 
-        divisor = serverClusterMeasurement.getAcPowerDivisor(Long.MAX_VALUE);
-        multiplier = serverClusterMeasurement.getAcPowerMultiplier(Long.MAX_VALUE);
-        if (divisor == null || multiplier == null) {
-            divisor = 1;
-            multiplier = 1;
-        }
+        determineDivisorAndMultiplier(serverClusterMeasurement);
```
Comment (cdjackson) [inline org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterMeasurementPower.java]: Why is this called here - this is at the end of the method, so surely nothing from this method is used after this?

File: org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterMeasurementRmsCurrent.java [hunk 3]
```diff
@@ -72,12 +72,8 @@ public boolean initializeDevice() {
             return false;
         }
 
-        divisor = serviceClusterMeasurement.getAcCurrentDivisor(Long.MAX_VALUE);
-        multiplier = serviceClusterMeasurement.getAcCurrentMultiplier(Long.MAX_VALUE);
-        if (divisor == null || multiplier == null) {
-            divisor = 1;
-            multiplier = 1;
-        }
+        determineDivisorAndMultiplier(serverClusterMeasurement);
```
Comment (cdjackson) [inline org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterMeasurementRmsCurrent.java]: Why do we need to call this here? It's right at the end of the initialisation method, so anything done there is not used.

File: org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterMeasurementRmsVoltage.java [hunk 4]
```diff
@@ -72,12 +72,7 @@ public boolean initializeDevice() {
             return false;
         }
 
-        divisor = serverClusterMeasurement.getAcVoltageDivisor(Long.MAX_VALUE);
-        multiplier = serverClusterMeasurement.getAcVoltageMultiplier(Long.MAX_VALUE);
-        if (divisor == null || multiplier == null) {
-            divisor = 1;
-            multiplier = 1;
-        }
+        determineDivisorAndMultiplier(serverClusterMeasurement);
```
Comment (cdjackson) [inline org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterMeasurementRmsVoltage.java]: Same comment - why is this called?

File: org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterColorTemperature.java (hunk 1 above)
Comment (TomTravaglino) [reply org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterColorTemperature.java]: Will remove it.

File: org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterMeasurementPower.java (hunk 2 above)
Comment (TomTravaglino) [reply org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterMeasurementPower.java]: Will remove it.

File: org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterMeasurementRmsCurrent.java (hunk 3 above)
Comment (TomTravaglino) [reply org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterMeasurementRmsCurrent.java]: Will remove it.

File: org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterMeasurementRmsVoltage.java (hunk 4 above)
Comment (TomTravaglino) [reply org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterMeasurementRmsVoltage.java]: Will remove it.

**Comment by TomTravaglino** [comment]
I updated the PR
